src/fft_simd.rs

```diff
@@ -214,7 +214,18 @@ impl FftSimd<c64> for Scalar {
 
     #[inline(always)]
     fn mul(self, a: c64, b: c64) -> c64 {
-        a * b
```

Comment:
can you explain the reason for this change ? Avoid the compiler choosing which operations to perform and in what order ?

Reply:
the reason is that the simd implementations use fused multiply add, so we need to do the same for scalar c64
the implementation in num-complex uses unfused operations

Reply:
ok !